docker-compose.yml

```
@@ -2,32 +2,32 @@ version: '3'
 
 services:
 
-  main-db:
-    image: postgres:14-alpine
-    container_name: main-db
+  grants-db:
+    image: postgres:16-alpine
```

```suggestion
    image: postgres:14-alpine
```
Looks like we are using postgres 14: https://github.com/HHS/simpler-grants-gov/blob/b8776797bd240d07145dd0e55401bbb4005d94c0/infra/modules/database/main.tf#L74 , I assume b/c that is what the version of RDS we are using supports: https://aws.amazon.com/about-aws/whats-new/2023/01/amazon-aurora-supports-postgresql-versions/